src/test/java/com/rocketden/main/api/UserTests.java

```diff
@@ -158,34 +158,34 @@ public void createNewUserNicknameContainsSpaces() throws Exception {
         assertEquals(ERROR.getResponse(), actual);
     }
 
-    // Status is provided as 404. Something missing.
-    // @Test
+    @Test
     public void deleteExistingUser() throws Exception {
```

Comment:
I don't think DeleteMapping is appropriate to use anymore even though it would be a nice way to differentiate between all the different requests. I explained under Alan's comment. This method looks messy but there really aren't many differences; other than changing Delete to Put I think I mainly just reordered things.

Reply:
Actually let me look more into how DeleteMapping works.